src/components/atoms/ChatMessageInfo/ChatMessageInfo.tsx

```diff
@@ -33,6 +33,14 @@ export const ChatMessageInfo: React.FC<ChatMessageInfoProps> = ({
     openUserProfileModal(author);
   }, [openUserProfileModal, author]);
 
+  const handleDelete = useCallback(
+    (e: SyntheticEvent) => {
+      e.stopPropagation();
```

Comment:
we try to avoid using `e.stopPropagation()`. we tend to use `e.target.closest()` instead where possible. (you can find the examples in the schedule - `ScheduleEvent` click)

Reply:
done 074df636ec32a2e30c4560e4f98709dc5cbd6227